DuckDuckGo/AppDelegate.swift

```diff
@@ -45,6 +45,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func applicationDidBecomeActive(_ application: UIApplication) {
         startMigration()
+        TrackerLoader.shared.updateTrackers()
```

Comment:
Just checked with Networking monitoring, first request uses 88kb of network, subsequent uses 0.5kb.